api/v1alpha2/conversion_test.go

```diff
@@ -242,4 +279,4 @@ func TestConvertMachineDeployment(t *testing.T) {
 			g.Expect(restored.Spec.Template.Spec.ClusterName).To(Equal(src.Spec.Template.Spec.ClusterName))
 		})
 	})
-}
+}
```

Comment:
Please don't remove the newline at the end of the file. Thanks.

Reply:
Done